test/fips.h

```diff
@@ -0,0 +1,7 @@
+#include <mls/crypto.h>
+
+bool
+fips();
+
+bool
+is_fips_approved(mls::CipherSuite::ID id);
```

Comment:
If we need these methods in `test`, let's just make them a public API of the `hpke` library.  We shouldn't be directly including OpenSSL headers outside of HPKE.

Reply:
This method is no longer needed. Initially, I needed it when I tested my changes on an Oracle Linux 8 with FIPS enabled. The OS came with OpenSSL 1.1.1k FIPS and it had a smaller set of FIPS approved algorithms compared to [OpenSSL 3.0 FIPS](https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp4282.pdf). So I had to use the above method to skip certain algorithm choices. Given that we are moving towards the later version of OpenSSL, I don't see there is need to skip any test so I think we can remove this method.